modules/tinymce/src/core/main/ts/selection/SelectionUtils.ts

```diff
@@ -110,8 +115,34 @@ const hasAnyRanges = (editor: Editor) => {
   return sel && sel.rangeCount > 0;
 };
 
+const runOnRanges = (editor: Editor, executor: (rng: Range, fake: boolean) => void) => {
```

Comment:
Is this just intended for Formatter styles or also table styles? Thinking of how some things like border/colours can be applied to to tds, trs, etc. Idk what getCellsFromEditor returns but I assume tds from the name, so wondered if it might miss some things/needs a comment to say what this does and doesn't apply to?

Reply:
Discussed on slack. There was some confusion about what this does, so I've added a comment to clarify.